### DISCUSSION TITLE
3.0.0-beta11

### DISCUSSION THREAD

**Discussion by Him188** [open-ani/animeko#168] [Announcements]
- 简化自动缓存设置的提示
- 支持自动创建缓存
   目前逻辑是每小时以及刚启动应用时检测一次, 每创建一个缓存延迟一分钟
- 增加配置载入时的动画 (#151)
- 增加删除缓存确认 (#153)
- 修复删除种子时可能崩溃的问题

 
----

**附件:**

- `.zip` 是包含 JRE 的解压即用的绿色版本；
- `.dmg`、`.deb`、`.rpm` 和 `.msi` 分别是 macOS、Debian、RedHat 和 Windows 平台的安装程序；
- `.exe` 是包含 JRE 的 Windows 程序，可以直接运行。
- `.jar` 是不包含 JRE 的最小程序，需要配合 JRE 11 及以上使用；
- `.apk` 是 Android 应用，支持的最低 Android 版本为 8.0；

注意，桌面平台仅支持 64 位操作系统。在使用 macOS 的安装程序时可能提示文件损坏，这是因为此版本应用没有签名，可以忽略错误继续使用。

也可扫描以下二维码下载 Android 版本：
![](https://github.com/Him188/ani/releases/download/v3.0.0-beta11/ani-3.0.0-beta11.apk.qrcode.png)

<hr /><em>This discussion was created from the release <a href='https://github.com/open-ani/ani/releases/tag/v3.0.0-beta11'>3.0.0-beta11</a>.</em>

**Comment by NeKoOuO** [comment]
這個版本崩潰問題好像有點多

**Reply by Him188** [reply]
什么时候会崩溃呢

**Reply by Him188** [reply]
我昨天发这个版人有点晕，写自动缓存忘了捕获异常，所以如果自动缓存怎么样失败了（比如网络错误）就会崩溃...

**Reply by NeKoOuO** [reply]
> 什么时候会崩溃呢

有時候就是正常情況就崩潰 不太清楚怎麼復現

**Reply by Him188** [reply]
那应该就是我昨天写的 bug, 待会发个新版
